lib/commands/promote.js

```diff
@@ -0,0 +1,119 @@
+'use strict';
+
+const chalk = require('chalk');
+const isMissingData = require('./validations').isMissingData;
+const max_promotions = 3;
+
+let elapsedInterval;
+let spamSeed = generateSeed()
```

Comment:
semicolon

Reply:
and const, unless you reassign.  Do you want to change the spam seed on each call?

Reply:
This is actually a misnomer from a quick copy paste I should have cleaned up, apologies. This is for a made up address not seed. I'm not sure if we should even generate made up addresses or just decide one constant address when promoting? Like we can just use CLIAPPPROMOTION99999999.... if we want.

Reply:
Here's my general rule of thumb.  Follow what the GUI wallet does.  I haven't taken a look at the code but it's an electron/javascript based app, so the code translates over pretty much directly.  Go take a peak there and see what they did.

Reply:
Thanks for the tip on this btw, will use that as a guide now.